tailwind.config.cjs

```diff
@@ -1,5 +1,6 @@
 /** @type {import('tailwindcss').Config} */
 const defaultTheme = require("tailwindcss/defaultTheme");
+const plugin = require("tailwindcss/plugin");
```

Comment:
Let's break this into another PR somewhere else. We can actually independently style the lists within the components for the page. Happy to help with the styling after (or before and we can rebase this)

Reply:
I'm under the impression we can't do this because of the way the markdown parsing is set up and we pass the rendered HTML as a prop.

We render the entire `body` key out of Markdown here: https://github.com/CircleCI-Public/Config.Tips/blob/main/src/pages/c/%5BconfigKind%5D/index.astro#L19

Then we retrieve it: https://github.com/CircleCI-Public/Config.Tips/blob/main/src/pages/c/%5BconfigKind%5D/index.astro#L31

And render it wholesale without any modification: https://github.com/CircleCI-Public/Config.Tips/blob/main/src/pages/c/%5BconfigKind%5D/index.astro#L57

Please let me know if this understanding is incomplete. To me, this is the simplest way to go about styling lists by default, but opting out where necessary.

Happy to break this up, and happy to figure out a better way to style lists!

Reply:
What we can do is create a custom renderer to attach a specific class to the components created with marked, then we can style them like normal.

https://github.com/CircleCI-Public/Config.Tips/blob/17a9568644b84a21a85b46626b969de10c40ceef/src/pages/c/%5BconfigKind%5D/%5Btip%5D.astro#L30

Reply:
Ah, that's cool. I'll back out the theme changes and break it into a second PR.

I figure we'll want to merge the renderer changes first, though.